include/sea_dsa/Graph.hh

```diff
@@ -529,36 +568,35 @@ private:
   accessed_types_type m_accessedTypes;
   /// destination of every offset/field
   links_type m_links;
-
   /// known size
   unsigned m_size;
-
   /// allocation sites for the node
-  typedef boost::container::flat_set<const llvm::Value *> AllocaSet;
-  AllocaSet m_alloca_sites;
-
+  ValueSet m_alloca_sites;
+  /// call sites for the node
+  ValueSet m_call_sites;
```

Comment:
I guess these are call sites that point to this node. This is not easy to maintain and I don't see the reasoning.
Seems simple to have a list of callsites at the level of the graph rather than partition the list by nodes that they point into